parser.go

```diff
@@ -275,7 +275,7 @@ func SetExcludedDirsAndFiles(excludes string) func(*Parser) {
 	}
 }
 
-// SetExcludedDirsAndFiles sets directories and files to be excluded when searching.
+// SetPackagePrefix sets directories and files to be excluded when searching.
```

Comment:
This comment is copied by accident, maybe we should remove it.

Reply:
It is required if the function is exportable.
Just correct the comment content,please.

Reply:
done

Reply:
I rebased to master by the way.